### ISSUE TITLE
Backspace default key binding makes the cursor jump to the beginning of the line

### ISSUE THREAD

**Issue by pwlmaciejewski** [facebook/draft-js#1971]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**

React 15.x
1. Go to https://jsfiddle.net/5vng6qrf/1/
2. Focus the editor and start typing something
3. Press Ctrl+Enter to clean the editor
4. Write something again
5. Hit backspace
6. Cursor jumps back to the beginning of the line

On React 16.x cursor also jumps to the beginning of the line but it additionally jumps back after hitting the first letter in step 4. Demo: https://jsfiddle.net/n31qsk5b/1/

**What is the expected behavior?**
Backspace should not break the editor

**Workarounds**
1) Do NOT return `getDefaultKeyBinding(e)` in custom key binding function
2) Force focus editor DOM node after cleaning the editor

**Which versions of Draft.js, and which browser / OS are affected by this issue? Did this work in previous versions of Draft.js?**
Draft v0.10.5
Chromium v71.0.3578.98

**Comment by claudiopro** [comment]
FWIW I can repro and notice that the cursor stays at the end of the first line when hitting Enter after step 4. Thanks for reporting this and providing a workaround @pwlmaciejewski !

**Comment by pwlmaciejewski** [comment]
@claudiopro yup, it seems that it affects Enter as well in the scenario you described. 
It may be related to https://github.com/facebook/draft-js/issues/1960.

**Comment by LovingGarlic** [comment]
> On React 16.x cursor also jumps to the beginning of the line but it additionally jumps back after hitting the first letter in step 4.

Also having this issue. The given workaround doesn't seem to work for me
